Source/SwiftLintFramework/Extensions/Configuration+Merging.swift

```diff
@@ -61,8 +61,8 @@ extension Configuration {
             return type(of: lhs.rule).description.identifier == type(of: rhs.rule).description.identifier
         }
 
-        fileprivate var hashValue: Int {
-            return type(of: rule).description.identifier.hashValue
+        public func hash(into hasher: inout Hasher) {
```

Comment:
This needs to be `private`.

Reply:
Will set to `fileprivate` as before, ok?

Reply:
👍